console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/DeviceAddDialog.java

```diff
@@ -107,7 +108,14 @@
     public DeviceAddDialog(GwtSession currentSession) {
         super(currentSession);
 
-        DialogUtils.resizeDialog(this, 550, 450);
+        DialogUtils.resizeDialog(this, 550, 500);
+    }
+
+    @Override
+    protected void onRender(Element parent, int pos) {
+        super.onRender(parent, pos);
+        bodyPanel.setAutoHeight(true);
+        setAutoHeight(true);
```

Comment:
Just a question.

Would make sense to move 

```
bodyPanel.setAutoHeight(true);
setAutoHeight(true);
```

to some of the parent Classes? 
To cover all Dialogs and not only the DeviceAdd one
Or does it causes other problems?

Reply:
Hi @Coduz. I can try to do this, but all dialogs will have auto height. Is that ok?  
_Regards._

Reply:
Hi @ct-anaalbic ,

well I don't actually know what are the consequences of that! 
Could you please try and see if that makes everything else ok, or it causes problems? 

If the second, we can just keep things as they are now.

Regards,

_Alberto_

Reply:
I tried to put this two line of codes into parent class, but that is not ok, because all dialogs have auto height, and problem with shadow is back, also. I think the best solution is to put this code into proper dialog, like above.

_Regards._